src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -363,6 +363,10 @@ public String getUploadManagerClass() {
         return getString("secor.upload.manager.class");
     }
 
+    public String getInnerUploadManagerClass() {
+        return getString("secor.inner.upload.manager.class");
```

Comment:
The way this reads makes getString feel like it's probably redundant.  Wouldn't we want the class name and not an interpretation of the class name?

Reply:
getString in this context is a function of the SecorConfig class that pulls the value out of the properties file.